Cargo.toml

```diff
@@ -10,6 +10,16 @@ license = "MIT OR Apache-2.0"
 categories = ["caching"]
 keywords = ["incremental", "memoization", "tracked", "constraints"]
 
+[features]
+default = [ "last_was_hit" ]
+last_was_hit = []
+
 [dependencies]
 comemo-macros = { version = "0.3.1", path = "macros" }
+hashbrown = "0.14.3"
+once_cell = "1.18.0"
+parking_lot = "0.12.1"
```

Comment:
Does parking lot make any performance difference? The lock poising is a bit of a pain, but if that's the only thing, maybe it's not worth to use it over std.